head-and-mouth/test_load.py

```diff
@@ -10,8 +10,13 @@
 
 import crdch_model as ccdh
 
+# The URI where the CRDCH YAML file is located.
+CRDCH_YAML_URI = 'https://raw.githubusercontent.com/cancerDHC/ccdhmodel/v1.1/model/schema/crdch_model.yaml'
```

Comment:
Chris M made a comment in the [LinkML slack recently](https://monarchinitiative.slack.com/archives/C01LDTVURH9/p1638228710114400?thread_ts=1638201048.109500&cid=C01LDTVURH9) about discouraging the use of GH raw URLs in code. I know that we're thinking about bundling the CRDC-H yaml file in the package. But if we wanted a PURL for the model file, we'd have to register that somewhere outside of GH, right? (Probably not worth it in this case.)

Reply:
I agree that a PURL is probably overkill for our needs, and it would be nice to bundle the CRDCH model YAML into the package directly. I've opened an issue to replace this URI by including a copy of the schema in the crdch_model package: https://github.com/cancerDHC/example-data/issues/45.